demo/2d_hand_demo.md

```diff
@@ -15,10 +15,12 @@ python demo/top_down_img_demo.py \
     ${MMPOSE_CONFIG_FILE} ${MMPOSE_CHECKPOINT_FILE} \
     --img-root ${IMG_ROOT} --json-file ${JSON_FILE} \
     --out-img-root ${OUTPUT_DIR} \
-    [--show --device ${GPU_ID}] \
+    [--show --device ${GPU_ID or CPU}] \
```

Comment:
How to interpret `or CPU`? `--device CPU`?

Reply:
Yes. I want to show that we can use cpu only mode by setting `--device=cpu`

Reply:
okay, does `CPU` vs `cpu` matter?